hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/AddressBookServiceImpl.java

```diff
@@ -31,6 +31,6 @@ public final class AddressBookServiceImpl implements AddressBookService {
     @Override
     public void registerSchemas(@NonNull SchemaRegistry registry) {
         registry.register(new V053AddressBookSchema());
-        registry.register(new V056AddressBookSchema());
+        registry.register(new V057AddressBookSchema());
```

Comment:
Should we rename or add new schema?

Reply:
The previous `V056AddressBookSchema` is actually deleted; so this PR is adding a new schema.